packages/devextreme/testing/tests/DevExpress.viz.core/translator2D.tests.js

```diff
@@ -798,6 +798,15 @@ QUnit.test('Translate. Update translator with business range with empty scale br
     assert.strictEqual(translator.translate(450), 1000);
 });
 
+QUnit.test('Translate. Scale breaks. Translator should handle division by zero (T1222629)', function(assert) {
+    const breaks = [{ from: -11.9, to: -10.3, cumulativeWidth: 5 }, { from: -10.3, to: 20.3, cumulativeWidth: 10 }];
```

Comment:
cant we pass breaks as an arg to createTranslatorWithScaleBreaks? like this const translator = createTranslatorWithScaleBreaks.call(this, { min: 200, max: 700, breaks: [] });